app/views/shared/_display_ad.html.erb

```diff
@@ -5,17 +5,22 @@
   data-context-type="<%= data_context_type %>"
   data-type-of="<%= display_ad.type_of %>">
   <div class="crayons-sponsorship__header relative">
-    <% if Page.exists?(slug: "billboards") && display_ad.organization_id? && display_ad.type_of == "community" %>
+    <% if display_ad.organization_id? && (display_ad.type_of == "community" || display_ad.type_of == "external") %>
       <img width="24" height="24" class="radius-default crayons-sponsorship__image" src="<%= display_ad.organization.profile_image_url_for(length: 64) %>" alt="profile" loading="lazy" />
-      <div class="crayons-sponsorship__title pl-1 fs-s"><%= display_ad.organization.name %></div>
+      <div class="crayons-sponsorship__title pl-1 fs-s fw-medium"><%= display_ad.organization.name %></div>
+      <% if display_ad.type_of == "external" %>
+        <span class="crayons-sponsorship__indicator c-indicator c-indicator--subtle c-indicator--round fs-xs fw-medium ml-1 py-1 px-2">PROMOTED</span>
```

Comment:
Optional suggestion: Maybe we can define this `PROMOTED` in en.yml and fr.yml files.

Reply:
Done ✅ thanks for the suggestion!